<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UiGpuItem</class>
 <widget class="QWidget" name="UiGpuItem">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1014</width>
    <height>222</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QLabel[type=&quot;key&quot;]{
	font-size: 10px;
	font-family: &quot;微软雅黑&quot;;
}

QLabel[type=&quot;value&quot;]{
	font: 75 20px &quot;微软雅黑&quot;;
	color: &quot;blue&quot;;
}</string>
  </property>
  <layout class="QGridLayout" name="gridLayout" rowstretch="0,1">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item row="0" column="3">
    <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="0,1">
     <item>
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>USED(M):</string>
       </property>
       <property name="type" stdset="0">
        <string>key</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lbUsed">
       <property name="text">
        <string/>
       </property>
       <property name="type" stdset="0">
        <string>value</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="0,1">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>NAME:</string>
       </property>
       <property name="type" stdset="0">
        <string>key</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lbName">
       <property name="text">
        <string/>
       </property>
       <property name="type" stdset="0">
        <string>value</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="2">
    <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="0,1">
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>FREE(M):</string>
       </property>
       <property name="type" stdset="0">
        <string>key</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lbFree">
       <property name="text">
        <string/>
       </property>
       <property name="type" stdset="0">
        <string>value</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,1">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>ID:</string>
       </property>
       <property name="type" stdset="0">
        <string>key</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lbId">
       <property name="text">
        <string/>
       </property>
       <property name="type" stdset="0">
        <string>value</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="4">
    <layout class="QHBoxLayout" name="horizontalLayout_5" stretch="0,1">
     <item>
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>TOTAL(M):</string>
       </property>
       <property name="type" stdset="0">
        <string>key</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lbTotal">
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="type" stdset="0">
        <string>value</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0" colspan="5">
    <widget class="QWidget" name="widget" native="true">
     <layout class="QGridLayout" name="gridLayout_3">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item row="0" column="0">
       <layout class="QGridLayout" name="grid">
        <property name="spacing">
         <number>0</number>
        </property>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
